scripts/update_assets/get_assets.py

```diff
@@ -0,0 +1,79 @@
+import argparse
+import csv
+import requests
+from urllib.parse import urljoin
+
+DEFAULT_API_URL = 'https://se-api.holmsecurity.com/v1/'
+
+
+def get_args():
+    parser = argparse.ArgumentParser(description='API key of the sc account')
+
+    parser.add_argument('-k',
+                        '--key-token',
+                        type=str,
+                        help='Token needed to access sc-dev',
+                        required=True)
+    parser.add_argument(
+        '-u',
+        '--url',
+        help=f"API URL to use, default is set to {DEFAULT_API_URL}",
+        default=DEFAULT_API_URL)
+    return parser.parse_args()
+
+
+def get_asset_request(args):
+    """
+    gets data from the endpoint and rewrites it to desired format.
+    """
+
+    url = urljoin(args.url, 'net-scans/assets')
+    headers = {"Authorization": f"TOKEN {args.key_token}"}
+    response = requests.get(url=url, headers=headers)
+    response.raise_for_status()
+    list_values = [v for v in response.json().values()]
+    assets_content = list_values[3]
+    list = []
+    list_new = []
+    for dicts in assets_content:
+        dict_new = {
+            key: dicts[key]
+            for key in dicts.keys()
+            & {'uuid', 'name', 'type', 'tags', 'ip', 'ip_range'}
+        }
+        list.append(dict_new)
+    for row in list:
+        dict_fields = {
+            'uuid': row['uuid'],
+            'name': row['name'],
+            'type': row['type'],
+            'tags': [],
+            'ip': row['ip'],
+            'ip_range': row['ip_range']
+        }
+        for elements in row['tags']:
+            dict_fields['tags'].append(elements['uuid'])
+        dict_joined = "|".join(dict_fields['tags'])
```

Comment:
The `dict_joined` variable is in fact a string. Maybe you can rename it to `joined_tags`?

Reply:
Done!